test/tools/ossfuzz/config/solidity.dict

```diff
@@ -0,0 +1,214 @@
+" address(this).balance "
+" address(this).call("
+" address(this).callcode("
+" address(this).delegatecall("
+" address(this).send("
+" address(this).transfer("
+" block.blockhash "
+" block.coinbase "
+" block.difficulty "
+" block.gaslimit "
+" block.number "
+" block.timestamp "
+" days"
+" ether"
+" finney"
+" gasleft() "
+" hours"
+" keccak256.gas() "
```

Comment:
I think this is too specific. Probably better to just provide `.gas(` and `.value(`

Reply:
How about providing both `.gas(` and `keccak256.gas(`? The second significantly increases probability of the fuzzer synthesizing specific `gas` calls over (irrelevant) others.